tools/subgraph-example/contracts/ExampleHTS.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+abstract contract ExampleHTS is ERC20 {
```

Comment:
nit: since in the next case you do NFT you can highlight this is fungible
```suggestion
abstract contract ExampleHTSFT is ERC20 {
```

Reply:
Renamed.